### ISSUE TITLE
Ahashpool "Do NOT use 24 hours actuals" advice

### ISSUE THREAD

**Issue by sepa85** [MultiPoolMiner/MultiPoolMiner#1180]
On the Ahashpool site the staff has placed this announcement:

> Announcement: [2018-02-04 22:49 UTC] Please do not use 24 hour actuals in your profit switching script. When chain split happens, the pool will disable coins, or even disable the entire algo, and if you are using the past 24 hours actuals algo profit ranking to pick algo, you will be wasting your hash power. Bul wark just had a chain split, we disabled Bulwark but there are still 700+ miners on Nist5 after an hour, this tells us at least 700 rigs are doing it wrong. You should only use current estimates in determining your algo. And if you don't want it to switch too often, just increase the interval time. Do NOT use 24 hours actuals. 

Is MPM "aligned" to this advice? Can be useful to improve something?

**Comment by aaronsace** [comment]
Yes, MultiPoolMiner does not use actuals and builds it's own stats.
